apps/src/templates/levelSummary/FreeResponseAIEvaluation.tsx

```diff
@@ -0,0 +1,128 @@
+import React, {useEffect, useState} from 'react';
+
+import {getChatCompletionMessage} from '@cdo/apps/aiTutor/chatApi';
+import Button from '@code-dot-org/component-library/button';
+import CollapsibleSection from '@cdo/apps/templates/CollapsibleSection';
+
+import SafeMarkdown from '../SafeMarkdown';
+
+import style from '@cdo/apps/levelbuilder/ai-iteration-tools/ai-tutor/ai-tutor-tester.module.scss';
+
+interface StudentResponse {
+  user_id: number;
```

Comment:
😢 this under_score casing is just part of how the data was already structured reworking to be camelCase feels like nice to have code clean up but I didn't want to go down that rabbit hole right now